minio/minio
# Avoid notification event for replicas

Creating notification events for replica creation
is not particularly useful to send as the notification
event generated at source already includes replication
completion events.

For applications using replica cluster as failover, avoiding
duplicate notifications for replica event will allow seamless
failover.

## Description

Needs https://github.com/minio/minio-go/pull/1460
## Motivation and Context


## How to test this PR?
Set up replication on a bucket using https://github.com/minio/minio/blob/master/docs/bucket/replication/README.md and turn on `mc watch target/bucket` to see notifications being generated on target

## Types of changes

## Checklist:
